src/include/daos/rpc.h

```diff
@@ -93,7 +93,7 @@ enum daos_rpc_type {
 };
 
 /** DAOS_TGT0_OFFSET is target 0's cart context offset */
-#define DAOS_TGT0_OFFSET		(1)
+#define DAOS_TGT0_OFFSET		(2)
```

Comment:
this is fixed value also, did not see how can it configurable (as mentioned in commit msg).

Reply:
I removed customization of SWIM context.